### DISCUSSION TITLE
MusicBrainz

### DISCUSSION THREAD

**Discussion by jo99jo** [enzo1982/freac#460] [Help & Support] [answered]
Per the fre:ac v1.1.7 latest news page, it is listed that:

"Added support for MusicBrainz metadata in ID3v2, MP4, Vorbis Comment, APEv2 and WMA tags"

So where is this?  I downloaded 1.1.7 but don't see MusicBrainz or anyway to enable it.

**Comment by enzo1982** [comment, marked as answer]
That line refers to commit enzo1982/BoCA@fc37195 which adds support for MusicBrainz metadata fields in tags.

What it means for users of fre:ac is that when converting files that have been tagged using e.g. MusicBrainz Picard, the special MusicBrainz tags (mostly MusicBrainz keys for the artist, album, disc, track etc.) will be preserved during the conversion. Before fre:ac 1.1.7, those would have been lost.

Actual MusicBrainz support will come to fre:ac later. I hope to make it happen this year.
